.github/workflows/native-image.yaml

```diff
@@ -48,7 +48,7 @@
     build-binary-windows:
       runs-on: windows-latest
       env:
-        GRAALVM_VERSION: 20.3.0.java11
+        GRAALVM_VERSION: 21.2.0.java8
```

Comment:
we want 22 + java17

Reply:
Will change that, I was trying to test to see if other versions would work